src/js/actions/transform.js

```diff
@@ -1132,8 +1133,8 @@ define(function (require, exports) {
             // all selected layers, but due to a recent bug, "move" event sends us the displacement
             // of layers from the changing (0,0) coordinates, which causes bugs like
             // getting (650,0) when the move was actually (-100, 0) for a 750 px wide layer
-            this.flux.actions.layers.resetBounds(nextDoc, nextDoc.layers.allSelected);
-        }.bind(this);
+            return this.flux.actions.layers.resetBounds(nextDoc, nextDoc.layers.allSelected);
+        }.bind(this), this);
```

Comment:
I'm registering my hunch that `bind(this)` is not necessary (but harmless) now that debounce will bind it within